app/src/main/java/com/example/pix/chat/fragments/ChatFragment.java

```diff
@@ -101,6 +102,12 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
             Toast.makeText(getContext(), "Error retrieving more chats", Toast.LENGTH_SHORT).show();
         }
 
+        ivProfile.setOnClickListener(view12 -> getParentFragmentManager().beginTransaction()
+                .setCustomAnimations(R.anim.slide_in_up, R.anim.slide_out_up)
+                .addToBackStack("stack")
```

Comment:
I actually removed this line in a future pull request. I attempted to add the current fragment (Chatting with a friend) to a back stack I just named "stack", then going to the profilefragment of this friend, and when Im done retrieving the chat fragment in my stack. I decided to change it but now that I think about it, I will try to do this again, since the way I have it now retrieves the previous fragment by making it a public static variable.